azure/iac/master-compliance-test.json

```diff
@@ -2586,7 +2586,7 @@
                             "id": "PR-AZR-PR-AZR-0112-ARM-ARM",
                             "eval": "data.rule.blobService",
                             "message": "data.rule.blobService_err",
-                            "remediationDescription": "",
+                            "remediationDescription": "Make sure StorageAccount has ( properties.encryption.services with [ name='blob' and enabled=true ] )",
```

Comment:
**See my validation message at line 2496 (for id PR-AZR-0109-ARM).** make the description format like this one.